pytest_fixtures/component/taxonomy.py

```diff
@@ -1,6 +1,7 @@
 # Content Component fixtures
 import pytest
 
+from manifester import Manifester
```

Comment:
We probably want to add manifester to requirements.txt?

Reply:
probably :)